src/main/java/com/minecolonies/coremod/colony/Colony.java

```diff
@@ -448,6 +502,7 @@ private boolean worldTickSlow()
     private boolean worldTickUnloaded()
     {
         updateChildTime();
+        checkForUnload();
```

Comment:
move this to worldTickSlow instead, here its only updated when the colony is not loaded

Reply:
It's in both. We need it in both, else it won't unload it when the player is not around. (offline or far away)

Reply:
oh true we want both